src/SearchParameters/index.js

```diff
@@ -89,21 +89,6 @@ function findArray(array, searchedValue) {
 function SearchParameters(newParameters) {
   var params = newParameters ? SearchParameters._parseNumbers(newParameters) : {};
 
-  /**
-   * Targeted index. This parameter is mandatory.
-   * @member {string}
-   */
-  this.index = params.index || '';
```

Comment:
just wondering if I understand this correctly: with this removal you can no longer read the values from the parameters directly, so `new SearchParameters().index === undefined` always?

Reply:
Yes you can if they are defined otherwise you get `undefined`. It changes nothing expect for `query` and `page`. `index` is required anyway so the default value was never used.